graphql/handler/transport/websocket_graphqlws.go

```diff
@@ -14,6 +14,7 @@ const (
 	graphqlwsConnectionInitMsg      = graphqlwsMessageType("connection_init")
 	graphqlwsConnectionTerminateMsg = graphqlwsMessageType("connection_terminate")
 	graphqlwsStartMsg               = graphqlwsMessageType("start")
+	graphqlwsStartAckMsg            = graphqlwsMessageType("start_ack")
```

Comment:
I don't see "start_ack" in the protocol mentioned in the comment. Maybe @jordanabderrachid (author of this file) can review it?

Reply:
Since everything else was working I was trying to just add one more message that AWS AppSync does. I will make a sub-protocol instead since as you said it isn't part of the spec for this protocol.